composer.json

```diff
@@ -9,8 +9,11 @@
         "composer/semver": "~1.2",
         "phpunit/php-timer": "^1.0.4",
         "phpunit/phpunit": "^6.0.13",
-        "symfony/console": "^3.2",
-        "symfony/process": "^3.2"
+        "symfony/console": "^3.4|^4.0",
+        "symfony/process": "^3.4|^4.0"
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^2.3.2"
```

Comment:
2.3.x does not support symfony 4, only the later versions do.

Reply:
Last 2.x version does support it.

BTW, you should not require this binary as a project dependency.

Reply:
This was already a dependency in the project, and indeed we're including the latest now.